analyzers/its/expected/akka.net/S1694-Akka.Remote-netstandard2.0.json

```diff
@@ -1,11 +1,5 @@
 {
   "Issues": [
-    {
-      "Id": "S1694",
-      "Message": "Convert this \u0027abstract\u0027 class to an interface.",
-      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/akka.net/src/core/Akka.Remote/FailureDetector.cs#L17",
```

Comment:
FP: the class cannot be converted to an interface, because in .NET Standard 2.0 interfaces cannot have [readonly static fields](https://github.com/SonarSource/sonar-dotnet/blob/fd7a9bea40fa21361bd8e05fcbba8365f3648307/analyzers/its/Projects/akka.net/src/core/Akka.Remote/FailureDetector.cs#L42).